src/modeler/AutoRest.Swagger/Validation/Core/Rule.cs

```diff
@@ -27,6 +27,11 @@ protected Rule()
         public virtual ValidationCategory ValidationCategory => ValidationCategory.None; // !!! implement me and make me abstract !!!
 
         /// <summary>
+        /// What kind of change implementing this rule can cause.
+        /// </summary>
+        public virtual ValidationChangesImpact ValidationChangesImpact => ValidationChangesImpact.None;
```

Comment:
Not really related to this change, but as we're at this, should we add another Rule property for "RuleInfo" or documentation, so we can provide a link to extra documentation?